src/plugins/intel_cpu/src/nodes/executors/acl/acl_lowp_fullyconnected.cpp

```diff
@@ -0,0 +1,145 @@
+#include "nodes/executors/debug_messages.hpp"
+#include "nodes/executors/implementation_utils.hpp"
+#include "nodes/executors/common/common_utils.hpp"
+#include "utils/debug_capabilities.h"
+
+namespace ov {
+namespace intel_cpu {
+
+static bool checkPostOps(const PostOps &postOps) {
+    if (postOps.empty()) {
+        return true;
+    }
+
+    if (postOps.size() != 1) {
+        return false;
+    }
+
+    const auto activation = std::dynamic_pointer_cast<ActivationPostOp>(postOps[0]);
+    return checkActivationLayerInfo(convertToEltwiseAlgorithm(activation->type()));
+}
+
+static void initFCAttrs(const FCAttrs &attrs,
+                        ACLTensorAttrs& aclTensorAttrs,
+                        ACLFCAttrs& aclfcAttrs,
+                        const MemoryArgs &memory,
+                        arm_compute::GEMMInfo& fullyConnectedLayerInfo,
+                        const PostOps &postOps) {
+    aclTensorAttrs.hasLayoutTypeNHWC = memory.at(ARG_SRC)->getDescPtr()->hasLayoutType(LayoutType::nspc);
+    aclfcAttrs.inputPrecision = memory.at(ARG_SRC)->getDescPtr()->getPrecision();
+    aclfcAttrs.weightsNonTransposed = attrs.weightsNonTransposed;
+
+    if (!postOps.empty()) {
+        auto activation = std::dynamic_pointer_cast<ActivationPostOp>(postOps[0]);
+        fullyConnectedLayerInfo.set_activation_info(getActivationLayerInfo(
+                convertToEltwiseAlgorithm(activation->type()),
+                activation->alpha(), activation->beta(), activation->gamma()));
+    }
+
+    if (memory.at(ARG_SRC)->getPrecision() != memory.at(ARG_WEI)->getPrecision()) {
+        aclfcAttrs.isConvertedWeights = true;
+    }
+}
+
+ACLLowpFullyConnectedExecutor::ACLLowpFullyConnectedExecutor(const FCAttrs &attrs,
+                                                             const PostOps &postOps,
+                                                             const MemoryArgs &memory,
+                                                             const ExecutorContext::CPtr& context) {
+    dequantizationScales = getDeQuantizedScales(memory);
+    initFCAttrs(attrs, aclTensorAttrs, aclfcAttrs, memory, gemmInfo, postOps);
+    packedWeights = acl_fc_executor::prepareWeightMemory(memory, context, attrs, aclfcAttrs, postOps, expectedWeightFormat, weiTensorInfo);
+}
+
+bool ACLLowpFullyConnectedExecutor::supports(const FCConfig &config) {
+    const auto src0 = srcType(config);
+    const auto src1 = weiType(config);
+    const auto dst = dstType(config);
+    if ((src0 != ov::element::i8) || (src1 != ov::element::i8) || (dst != ov::element::f32)) {
+        return false;
+    }
+
+    VERIFY(checkPostOps(config.postOps), UNSUPPORTED_TYPE_OF_POSTOPS);
+    VERIFY(one_of(srcRank(config), 2U, 3U, 4U), UNSUPPORTED_SRC_RANK);
+    VERIFY(one_of(weiRank(config), 2U, 3U, 4U), UNSUPPORTED_WEI_RANK);
+    //VERIFY(static_cast<FCAttrs>(config.attrs).dequantizationScales.size() <= 1, UNSUPPORTED_PER_CHANNEL_QUANTIZATION);
```

Comment:
Why it is commented out?

Reply:
This check should be removed (I removed it).
DQ scales is not FC attribute anymore, DQ scales could be retrieved from `MemoryArgs`.

Reply:
I bet we just need to check the shape of corresponding input. If it is not scalar - unsupported case

Reply:
Implemented the check here: https://github.com/openvinotoolkit/openvino/pull/25171/files#diff-1f8e6cb50710138cdb232d82eec8726630df1b79cc840fa4f0d60a9704c30b2bR404-R407